website/docs/docs/build/join-logic.md

```diff
@@ -0,0 +1,153 @@
+
+```yaml 
+todo: update syntzx
+mf query --metrics average_purchase_price --dimensions metric_time,user_id__type 
+```
+
+:::tip Add prefix entity to dimensions with same name 
+To prevent the creation of ambiguous join paths, use a dimension that includes the prefix entity to disambiguate its origin if you have multiple dimensions with the same name. You can view the fully scoped dimension name by running the `mf list-metrics` command in the CLI.
+:::
+
+## Multi-hop joins
+
+MetricFlow allows users to join measures and dimensions across a graph of entities, which we refer to as a 'multi-hop join.' This is because users can move from one table to another like a 'hop' within a graph.
+
+Here's an example schema for reference:
+
+![Multi-Hop-Join](/img/docs/building-a-dbt-project/multihop-diagram.png)
+
+Notice how this schema can be translated into the three MetricFlow semantic models below to create the metric 'Average purchase price by country' using the `purchase_price` measure from the sales table and the `country_name` dimension from the `country_dim` table.
+
+```yaml
+semantic_models:
+  - name: sales
+	entities:
+        - name: id
+          type: primary
+        - name: user_id
+          type: foreign
+	measures:
+        - name: average_purchase_price
+          agg: avg
+          expr: purchase_price
+          create_metric: true
+	dimensions:
+        - name: ds
+          type: time
+          type_params:
+            is_primary: true
+  -	name: user_signup
+	entities:
+        - name: user_id
+          type: primary
+        - name: country_id
+          type: Unique
+	dimensions:
+        - name: signup_date
+          type: time
+          type_params:
+            is_primary: true
+  -	name: country_dim
+	entities:
+        - name: country_id
+          type: primary
+	dimensions:
+        - name: country_name
+          type: categorical
+```
+
+### Query multi-hop joins
+
+If you want to query dimensions _without_ a multi-hop join involved, use the dimension name with a double underscore (or dunder) prefix and the entity name. For dimensions retrieved by a multi-hop join, you can use multiple sets of double underscored entities since you are hopping across multiple semantic models.
+
+For example, if you want to see the `average_purchase_price` metric split by the `country_name` dimension, MetricFlow will involve a multi-hop based on the schema specified in the semantic model:
+
+1. From semantic model `sales` to semantic model `user_signup` through the `user_id` entity.
+
+2. From semantic model `user_signup` to semantic model `country_dim` through the `country_id` entity to obtain the desired final dimension of `country_name`. 
+
+To query this multi-hop join, use the dimension name `user_id__country_id__country_name` with two sets of double underscores to reflect the two entities involved (`user_id` and `country_id`):
```

Comment:
Again, we should update the multi dunder path section because I don't think we'll be implemetning this pattern anymore